src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -2185,6 +2185,18 @@ internal protected virtual CSharpSyntaxNode GetBindableSyntaxNode(CSharpSyntaxNo
                 }
             }
 
+            switch (node.Kind())
```

Comment:
>switch (node.Kind()) [](start = 12, length = 20)

It feels like this is not the right place for the adjustment. The nodes have semantic meaning without the parent.

Reply:
I will be reverting this part of the PR after more discussion with Jason and Julien offline. This is just not something the IDE should expect to do and whatever answer they get back is what they get bakc.

---
In reply to: [366363378](https://github.com/dotnet/roslyn/pull/40950#discussion_r366363378) [](ancestors = 366363378)